### PR TITLE
Add Activate app

### LINKED ISSUE #360: Add Activate App endpoint
This issue is raised to track and maintain the dot-net-client on par with other language clients on API's. Hope to solve issues like
https://github.com/appium/appium-dotnet-driver/issues/335

Here are some of API's missing, feel free to pick and work on

- [ ] GET_PERFORMANCE_DATA
- [x] ACTIVATE_APP
- [ ] GET_DISPLAY_DENSITY
- [ ] GET_SYSTEM_BARS
- [ ] IS_KEYBOARD_SHOWN
- [ ] SEND_SMS
- [ ] GSM_CALL
- [ ] GSM_SIGNAL
- [ ] GSM_VOICE
- [ ] GET_ALLSESSION

Please feel free to suggest, if I've missed out any of the API's and will be glad to add it to this list.

### PR DESCRIPTION
**Pull request by manoj9788**
## Change list

Please provide briefly described change list which are you going to propose. 
 
## Types of changes

What types of changes are you proposing/introducing to .NET client?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [X] New feature (non-breaking change which adds value to the project)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Documentation
- [ ] Have you proposed a file change/ PR with appium to update documentation? 
## Integration tests
- [ ] Have you provided integration tests to pass against the beta version of appium? (for Bugfix or New feature)

## Details
Closes #360 
Please provide more details about changes if it is necessary. If there are new features you can provide code samples which show the way they
work and possible use cases. Also you can create [gists](https://gist.github.com) with pasted C# code samples or put them here using markdown. 
About markdown please read [Mastering markdown](https://guides.github.com/features/mastering-markdown/) and [Writing on GitHub](https://help.github.com/categories/writing-on-github/)

### CODE REVIEW

File: src/Appium.Net/Appium/Interfaces/IInteractsWithApps.cs [hunk 1]
```diff
@@ -59,6 +59,12 @@ public interface IInteractsWithApps : IExecuteMethod
         /// <param name="appPath">a string containing the id of the app.</param>
         void RemoveApp(string appId);
 
+        /// <summary>
+        /// Activates an App.
```
Comment (mykola-mokhnach) [inline src/Appium.Net/Appium/Interfaces/IInteractsWithApps.cs]: Activates the given app by moving to foreground if it is running in the background or starting it if it is not running yet

Comment (mykola-mokhnach) [reply src/Appium.Net/Appium/Interfaces/IInteractsWithApps.cs]: An error will be thrown if the app with the given identifier is not present on the device under test

**Comment by akinsolb** [comment]
@manoj9788 Can you update the changelog doc in the root of the project? I've merge once the documentation is updated

**Comment by manoj9788** [comment]
@akinsolb Done.

**Pull request merged**